crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -85,28 +85,55 @@ pub struct BinnedRenderPhase<BPI>
 where
     BPI: BinnedPhaseItem,
 {
-    /// A list of `BinKey`s for batchable items.
+    /// A list of `BatchSetKey`s for batchable, multidrawable items.
     ///
     /// These are accumulated in `queue_material_meshes` and then sorted in
     /// `batch_and_prepare_binned_render_phase`.
-    pub batchable_mesh_keys: Vec<BPI::BinKey>,
+    pub multidrawable_mesh_keys: Vec<BPI::BatchSetKey>,
 
-    /// The batchable bins themselves.
+    /// The multidrawable bins themselves.
     ///
-    /// Each bin corresponds to a single batch set. For unbatchable entities,
-    /// prefer `unbatchable_values` instead.
-    pub batchable_mesh_values: HashMap<BPI::BinKey, RenderBin>,
+    /// Each batch set key maps to a *batch set*, which in this case is a set of
+    /// meshes that can be drawn together in one multidraw call. Each batch set
+    /// is subdivided into *bins*, each of which represents a particular mesh.
+    /// Each bin contains the entity IDs of instances of that mesh.
+    ///
+    /// So, for example, if there are two cubes and a sphere present in the
+    /// scene, we would generally have one batch set containing two bins,
+    /// assuming that the cubes and sphere meshes are allocated together and use
+    /// the same pipeline. The first bin, corresponding to the cubes, will have
+    /// two entities in it. The second bin, corresponding to the sphere, will
+    /// have one entity in it.
+    pub multidrawable_mesh_values: HashMap<BPI::BatchSetKey, HashMap<BPI::BinKey, RenderBin>>,
+
+    /// A list of `BinKey`s for batchable items that aren't multidrawable.
+    ///
+    /// These are accumulated in `queue_material_meshes` and then sorted in
+    /// `batch_and_prepare_binned_render_phase`.
```

Comment:
I think this and the comment above are incorrect and should be `sort_binned_render_phase`? I was confused looking for the sort in `batch_and_prepare_binned_render_phase`.

Reply:
Indeed you're right, that comment is wrong! Fixed.